core/embed/models/T3B1/vendorheader/vendor_unsafe.json

```diff
@@ -0,0 +1,22 @@
+{
+    "header_len": 512,
+    "text": "UNSAFE, DO NOT USE!",
+    "hw_model": "T3B1",
+    "expiry": 0,
+    "version": [0, 0],
+    "sig_m": 2,
+    "trust": {
+        "_reserved": 0,
+        "_dont_provide_secret": false,
```

Comment:
dtto here? don't remember how this works for UNSAFE header, but in any case "don't provide secret" should be true.

Reply:
dropped https://github.com/trezor/trezor-firmware/commit/a80f055bcf5c718cb0cb69f1b6203a005f35f22b